src/test/java/Homework21.java

```diff
@@ -0,0 +1,140 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework21 extends BaseTest {
+
+    String myNewPlaylist = "My New Playlist";
+    String newPlaylistTitle = "Updated " + myNewPlaylist;
+
+    @Test(priority = 1, dataProvider = "User Credentials", description = "Create new playlist with specific name")
+    public void createPlaylist(String email, String password) {
+        loginToKoelWithDataProvider(email, password);
+
+        createPlaylist(myNewPlaylist);
+        String expectedMessage = "Created playlist \"" + myNewPlaylist + ".\"";
+        Assert.assertEquals(verifyNotificationMessage(), expectedMessage);
+    }
+
+    @Test(priority = 2, dataProvider = "User Credentials", description = "Add few songs to newly created playlist with specific name")
+    public void addSongsPlaylist(String email, String password) {
+        loginToKoelWithDataProvider(email, password);
+
+        String expectedMsgForSongAdded = "Added 1 song into \"" + myNewPlaylist + ".\"";
+
+        addSongToPlayList("Song");
+        Assert.assertEquals(verifyNotificationMessage(), expectedMsgForSongAdded);
+
+        addSongToPlayList("Reactor");
+        Assert.assertEquals(verifyNotificationMessage(), expectedMsgForSongAdded);
+        //addSongToPlayList("Take");
+        //Assert.assertEquals(verifyNotificationMessage(), expectedMsgForSongAdded);
+        //wait.until(ExpectedConditions.visibilityOf(notification));
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.